geocode/ngeso.py

```diff
@@ -169,17 +254,16 @@ def reverse_geocode_gsp(self,
         -----
         Return format needs some work, maybe switch to DataFrames in future release.
         """
-        logging.debug("Reverse geocoding %s latlons to 20220314 GSP", len(latlons))
-        if self.gsp_regions is None:
```

Comment:
I think this `if` condition should still be kept - it enables `self.gsp_regions_dict` to also be cached in-memory